libs/core/src/lib/carousel/carousel.component.html

```diff
@@ -40,7 +40,9 @@
             <ng-container *ngTemplateOutlet="buttonLeft"></ng-container>
         </ng-container>
 
-        <ng-container *ngTemplateOutlet="pageIndicators"></ng-container>
+        <ng-container *ngIf="_showPageIndicators">
+            <ng-container *ngTemplateOutlet="pageIndicators"></ng-container>
+        </ng-container>
```

Comment:
```suggestion
        <ng-container *ngIf="_showPageIndicators" [ngTemplateOutlet]="pageIndicators">
        </ng-container>
```
you can do it like this, just a bit simpler